config/jest/jestConfig.js

```diff
@@ -1,25 +1,27 @@
+const escapeRegex = require('escape-string-regexp');
 const { paths } = require('../../context');
+const slash = '[/\\\\]'; // Cross-platform path delimiter regex
```

Comment:
Could we use `path.sep` for this use case? Not sure how it copes inside a regex/pattern like this.

Reply:
Spoke about this, decided to leave it for now.